lib/wibox/drawable.lua

```diff
@@ -21,7 +21,7 @@ local surface = require("gears.surface")
 local timer = require("gears.timer")
 local grect =  require("gears.geometry").rectangle
 local matrix = require("gears.matrix")
-local hierarchy = require("wibox.hierarchy")
+local ghierarchy = require("wibox.hierarchy")
```

Comment:
What is the reasoning behind the name, "Global hierarchy"?

Reply:
Opps, I was thinking it was in gears, but it's indeed in wibox. Fixed